installing/upgrade/upgrade-to-r3.0.md

```diff
@@ -100,7 +100,7 @@ Now, when you have dom0 upgraded, you can install new templates from Qubes R3.0
 Upgrading template on already upgraded dom0
 -------------------------------------------
 
-When for some reason you did not upgraded all the templates and standalone VMs before upgrading dom0, you can still do this, but it will be somehow more complicated. This can be a case when you restore backup done on Qubes R2.
+If for some reason you did not upgrade all the templates and standalone VMs before upgrading dom0, you can still do this, but it will be somewhat more complicated. This can be the case when you restore backup done on Qubes R2.
```

Comment:
perhaps taking out somewhat altogether as it doesn't really add much. "it will be more complicated' or 'it will **probably** be more complicated' or 'it **may** be more complicated' (maybe the user doesn't find it complicated at all)